main.go

```diff
@@ -172,7 +173,7 @@ func registerDynamicCapabilities(port string, log logger.Handler) {
 
 func registerWorkloads(port string, log logger.Handler) {
 	log.Info("Getting crd names from repository for component generation...")
-	names, err := config.GetFileNames("linkerd", "linkerd2", "charts/linkerd2/templates")
+	names, err := config.GetFileNames("linkerd", "linkerd2", "charts/linkerd-crds/templates")
```

Comment:
I have a suggestion here.  Could we list these parameters to a separate file like `variable.go` or `const.go` to list them as `variable`?